can/interfaces/ixxat/canlib.py

```diff
@@ -638,8 +638,13 @@ def __init__(
         )
 
         log.info(
-            "Initialising Channel %d with the following parameters:  \n%s\n%s",
+            "Initialising Channel %d with the following parameters:\n"
```

Comment:
Suggest:
```
            "Initialising Channel %d with the following parameters:\n"
            "bOpMode=%d, bExMode=%d\n"
            "pBtpSDR=%s\n"
            "pBtpFDR=%s"
```
or the log line gets a bit squashed up and it is not cleare where the 'None' comes from on a non-FD bus: 

![image](https://github.com/MattWoodhead/python-can/assets/26529943/a4b6f8a6-b0bc-440f-aa5d-f8759f54aefe)

Reply:
fixed

Reply:
pBtpFDR can be set to a NULL-pointer, when CAN FD is not used. That's why i set it to None.